protobuf/proto/common.proto

```diff
@@ -238,6 +238,8 @@ message SecretList { repeated UniqueKeySecretValue data = 1000; }
 
 message ListSecretsResponse {
   string prefix = 1;
-  string publicKey = 2;
-  repeated string keys = 3;
+  string name = 2;
+  string publicKey = 3;
```

Comment:
Not sure why this was originally here :D Is this still required? @nandor-magyar

Reply:
The intent was to store the public key alongside the secret, we can decide if it is still valid or the node has a new public key.